package.json

```diff
@@ -17,7 +17,7 @@
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.5",
     "ember-ajax": "^2.4.1",
-    "ember-cli": "github:ember-cli/ember-cli#canary",
+    "ember-cli": "github:ember-cli/ember-cli#beta",
```

Comment:
lets use the tag actually

current beta branch is ~ 2.12, but sometime tomorrow we will branch master to beta and then this will be broken again